scripts/get_servers.ts

```diff
@@ -72,6 +73,6 @@ async function filter<T>(arr: Array<T>, fn: (input: T) => Promise<boolean>): Pro
     // Server healthy.
     return true;
   });
-
+})().finally(() => {
```

Comment:
let's not mix promise api with async-await - if you'd like to use finally then go with try-catch-finally https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#syntax but I would actually just use catch block here not to overcomplicate things

Reply:
I added the `finally` here to fix the new lint error.

```
/Users/marcin/Sync/Repos/github.com/SkynetLabs/skynet-js-2/scripts/get_servers.ts
  47:1  warning  Promises must be handled appropriately or explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
```

`try-finally` doesn't work to fix that because the outer `async` block is still unhandled.